docs/Options.md

```diff
@@ -405,6 +406,20 @@ The environment variable `COMPOSE_PATH_SEPARATOR` is supported too. For more inf
 | `SPACESHIP_DOCKER_COLOR` | `cyan` | Color of Docker section |
 | `SPACESHIP_DOCKER_VERBOSE` | `false` | Show complete Docker version |
 
+### Ansible (`ansible`)
+
+Ansible section is shown only in directories that contain valid ansible playbooks or `[.]ansible.cfg` files. Please be aware
+that switching on the option to display the ansible version causes some delay.
+
+| Variable | Default | Meaning |
+| :------- | :-----: | ------- |
+| `SPACESHIP_ANSIBLE_SHOW` | `false` | Show current Ansible version |
```

Comment:
I think it's true in the actual implementation.
```suggestion
| `SPACESHIP_ANSIBLE_SHOW` | `true` | Show current Ansible version |
```

Reply:
```
| `SPACESHIP_ANSIBLE_SHOW` | `true` | Show Ansible section |
| `SPACESHIP_ANSIBLE_SHOW_VERSION` | `false` | Show current Ansible version |
```

Reply:
so now the first line says "Ansible section" and it is actually set `true` by default